### PR TITLE
Change access level of styleManager in CarPlay

### PR DESCRIPTION
**Pull request by OttyLab**
### Implementation

Change access level of StyleManager [here](https://github.com/mapbox/mapbox-navigation-ios/blob/v2.0.0-beta.15/Sources/MapboxNavigation/CarPlayNavigationViewController.swift#L84) and [here](https://github.com/mapbox/mapbox-navigation-ios/blob/v2.0.0-beta.15/Sources/MapboxNavigation/CarPlayMapViewController.swift#L14) that enables change the style dynamically as same as [`NavigationViewController`](https://github.com/mapbox/mapbox-navigation-ios/blob/v2.0.0-beta.15/Sources/MapboxNavigation/NavigationViewController.swift#L288).

### Screenshots or Gifs

No screenshots

### CODE REVIEW

**Comment by MaximAlien** [comment]
Thanks @OttyLab. Your change looks good to me. Can you please also update changelog to mention your changes, as they modify public API?

File: Sources/MapboxNavigation/CarPlayMapViewController.swift [hunk 1]
```diff
@@ -11,7 +11,12 @@ import CarPlay
 @available(iOS 12.0, *)
 public class CarPlayMapViewController: UIViewController {
     
-    var styleManager: StyleManager?
+    /**
+     Controls the styling of NavigationViewController and its components.
```
Comment (MaximAlien) [inline Sources/MapboxNavigation/CarPlayMapViewController.swift:15]:
```suggestion
     Controls the styling of CarPlayMapViewController and its components.
```

File: Sources/MapboxNavigation/CarPlayNavigationViewController.swift [hunk 2]
```diff
@@ -77,12 +77,18 @@ public class CarPlayNavigationViewController: UIViewController {
         }
     }
     
+    /**
+     Controls the styling of NavigationViewController and its components.
```
Comment (MaximAlien) [inline Sources/MapboxNavigation/CarPlayNavigationViewController.swift:81]:
```suggestion
     Controls the styling of CarPlayNavigationViewController and its components.
```

File: Sources/MapboxNavigation/CarPlayMapViewController.swift (hunk 1 above)
Comment (OttyLab) [reply Sources/MapboxNavigation/CarPlayMapViewController.swift]: Thanks, fixed

File: Sources/MapboxNavigation/CarPlayNavigationViewController.swift (hunk 2 above)
Comment (OttyLab) [reply Sources/MapboxNavigation/CarPlayNavigationViewController.swift]: Thanks, fixed

**Comment by OttyLab** [comment]
changelog has been updated

**Review by MaximAlien — APPROVED** [review]

**Pull request merged**